aptos-move/aptos-vm/src/natives.rs

```diff
@@ -54,7 +54,8 @@ struct AptosBlankStorage {
 impl AptosBlankStorage {
     pub fn new() -> Self {
         Self {
-            counter: AtomicU32::new(0),
+            // Put some recognizable number, to easily spot missed exchanges
+            counter: AtomicU32::new(55551111),
```

Comment:
Same here, just so that we don't have to subtract 11, although doesn't mater too much